lib/src/util/rem_util.dart

```diff
@@ -36,46 +36,66 @@ var _changeSensor;
 @visibleForTesting
 dynamic get changeSensor => _changeSensor;
 
+bool _shouldStillMountRemChangeSensor = false;
 Element _changeSensorMountNode;
 @visibleForTesting
 Element get changeSensorMountNode => _changeSensorMountNode;
 
-void _initRemChangeSensor() {
```

Comment:
Changes in this PR are best viewed with whitespace-agnostic diffs: https://github.com/Workiva/over_react/pull/197/files?w=1